src/metpy/interpolate/points.py

```diff
@@ -345,7 +344,7 @@ def interpolate_to_points(points, values, xi, interp_type='linear', minimum_neig
     elif interp_type == 'natural_neighbor':
         return natural_neighbor_to_points(points, values, xi)
 
-    # If this is Barnes/Cressman, determine search_radios and hand it along to
+    # If this is Barnes/Cressman, determine search_radius and hand it along to
```

Comment:
Definitely prefer `search_radios` myself 🤪

Reply:
Since it's in code, the first thing that comes to mind is a "radio button" too, which is really confusing.